src/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```

Comment:
We don't need a nuget.config file I don't think

Reply:
Not sure how this got in here.  It has been sneakily creating one without me knowing.  `git add .` is not my friend.